lib/src/common/observer_controller.dart

```diff
@@ -277,21 +277,23 @@ mixin ObserverControllerForScroll on ObserverControllerForInfo {
   /// Jump to the specified index position without animation.
   ///
   /// If the height of the child widget and the height of the separator are
-  /// fixed, please pass the [isFixedHeight] parameter.
+  /// fixed, please pass the [isFixedHeight] parameter and the
+  /// [renderSliverType] parameter.
```

Comment:
It is recommended to explain the default handling when renderSliverType is not passed.

Reply:
copy that